store/localcachelayer/layer.go

```diff
@@ -33,6 +36,8 @@ type LocalCacheStore struct {
 	roleCache     *utils.Cache
 	scheme        LocalCacheSchemeStore
 	schemeCache   *utils.Cache
+	team          LocalCacheTeamStore
+	teamCache     *utils.Cache
```

Comment:
You should preserve the name of the cache here. It is not one single cache per store, is one cache per need for each store. In this case the cache is `teamAllTeamIdsForUserCache`.